project/src/main/java/com/google/sps/data/Popular.java

```diff
@@ -30,12 +30,12 @@
   Query query = new Query("Popular");
 
   // Store restaurant's popularity score (# times it has been 'favorited') in Datastore
-  public void addToPopularList(String placeID) {
+  public void addToPopularList(String placeID, String placeName) {
     // Load restaurants'popularity scores from Datastore to check if place already exists in popular list
     boolean alreadyExists = false;
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
-        String place = (String) entity.getProperty("place");
+        String place = (String) entity.getProperty("placeID");
         if (place.equals(placeID)) {
             alreadyExists = true;
```

Comment:
Instead of doing this, why don't we use Query.Filter so we don't have to process it on the result?

https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/Query.html?hl=en#setfilter